src/domain/reservation/manage/filters/ManageReservationsFilters.js

```diff
@@ -222,6 +241,27 @@ class ManageReservationsFilters extends React.Component {
               )}
             </Col>
           </Row>
+          <Row>
+            <Col md={6}>
+              <SelectField
+                id="reservationDownload"
+                label={t('ManageReservationsFilters.downloadReservationData.title')}
+                onChange={item => this.onReservationDownloadOptionChange(item.value)}
+                options={this.getReservationDownloadOptions()}
+                value={reservationDownloadType}
+              />
+            </Col>
+            <Col md={6}>
+              <Button
+                bsStyle="default"
+                className="app-ManageReservationsFilters__downloadButton"
+                key="reservation-download"
+                onClick={() => this.props.onReservationDownload(reservationDownloadType)}
```

Comment:
Why are you assigning the value to a const instead of just using this.state here?

Reply:
IMO it makes code a bit more readable and also that I need to use this variable in 2 places.